package.json

```diff
@@ -97,6 +97,7 @@
   },
   "dependencies": {
     "choo": "6.13.0",
+    "debug": "https://github.com/visionmedia/debug/tarball/207a6a2d53507ec9dd57c94c46cc7d3dd272306d/debug.tar.gz",
```

Comment:
non version number?

Reply:
Yeah, fix for webextension context (https://github.com/visionmedia/debug/pull/476) was not released yet.